packages/obonode/obojobo-chunks-multiple-choice-assessment/MCChoice/editor-component.js

```diff
@@ -65,10 +68,51 @@ class MCChoice extends React.Component {
 		)
 	}
 
+	duplicateNode() {
```

Comment:
This code seems like it's copied from oboeditor/components/node - Would it be possible to use those methods somehow? Or pull these functions into a util class which both this and Node could use?

Reply:
One possible solution is to move `duplicateNode` to the MoreInfoBox. We only need to pass in `element`. What do you think? 
We can do the same for a few other functions.

Reply:
That could work, I think I'd prefer to keep it in the base node class, but since we're only using it here I'd be fine with moving it.